packages/data-service-generator/src/server/static/src/prisma/prisma.service.ts

```diff
@@ -6,10 +6,4 @@ export class PrismaService extends PrismaClient implements OnModuleInit {
   async onModuleInit() {
     await this.$connect();
   }
-
-  async enableShutdownHooks(app: INestApplication) {
-    this.$on("beforeExit", async () => {
-      await app.close();
-    });
-  }
```

Comment:
@overbit why was this removed? Is it just no longer needed in Prisma 5?

Reply:
Not supported anymore and it is redundant since prisma already managed this